tools/apkanalyser/result/apk-analyser-result.xsl

```diff
@@ -52,7 +56,7 @@
       #note, #dt { font-style: italic; margin-right: 6px; }
       .hili { color: rgba(211, 84, 0,1.0); }
     </style>
-    <script src='../../resources/webrunner/jquery-1.10.2.min.js'></script>
+    <script src='https://code.jquery.com/jquery-2.1.3.min.js'></script>
```

Comment:
Though have no reason in commit message why make this change, I accept it as-is. Thanks.